### PR TITLE
add surpport for campaign 2.0 plus

### PR DESCRIPTION
**Pull request by jerry-yu**
add surpport for campaign 2.0 plus

### CODE REVIEW

File: pallets/credit/src/lib.rs [hunk 1]
```diff
@@ -439,13 +440,8 @@ pub mod pallet {
                     Self::_update_credit_setting(setting);
                 }
 
-                CampaignIdSwitch::<T>::insert(0, 0);
```
Comment (xcaptain) [inline pallets/credit/src/lib.rs:442]: `CampaignIdSwitch` not used anymore? how about remove related code.

Comment (jerry-yu) [reply pallets/credit/src/lib.rs:442]: check the code , It is used

Comment (xcaptain) [reply pallets/credit/src/lib.rs:442]: I only see `CampaignIdSwitch::insert`, any other place will read this storage?

Comment (jerry-yu) [reply pallets/credit/src/lib.rs:442]: check the read function of CampaignIdSwitch : fn campaign_id_switch

**Review by xcaptain — APPROVED** [review]

**Pull request merged**